test/Header/SetCookieTest.php

```diff
@@ -425,16 +425,6 @@ public function testPreventsCRLFAttackViaFromString()
         $header = SetCookie::fromString("Set-Cookie: leo_auth_token=example;\r\n\r\nevilContent");
     }
 
-    /**
```

Comment:
Don't remove the test, Instead assert the value is valid. If not regressions are welcome :)